README.md

```diff
@@ -42,15 +42,15 @@ Benefits
 
 - Requires minimal skills to get started (Just HTML)
 - Creates accessible prototypes
-- Once learned,then it provides Experience of much faster than any drag-and-drop tool.
-- Create dynamic designs that flex & change depending on the width of the screen
+- Once learned,it provides Experience a much faster prototyping experience than visual drag-and-drop tool.
+- Create dynamic designs that flex and change depending on the width of the screen
 - Create designs that interact and behave exactly as deployed
 - View desktop and mobile versions of a design at the same time
 - It Makes design decisions based on the actual browser behavior, rather than just assumptions in our designs. Think of it as deciding in the browser.
 - Allows us to test early and get feedback on reality—not assumptions from abstracted tool output. As soon as a single piece of design is in the browser, it can be tested.
 - Sets expectations early and allows the design to grow iteratively and organically.
-- The Code used in prototypes can serve as a basis for a production ready code.
-- Ultimately, it allows us to put content over form, embrace the fluidity of the web and the relocation of devices, and It focuses Specially on System about design components instead of pages.
+- Code used in prototypes can serve as a basis for production ready code.
+- Ultimately, it allows us to put content over form, embrace the fluidity of the web and the diaspora of devices, and It focuses Specially on System about design components instead of pages.
```

Comment:
The latter part of this sentence is awkward. I suggest just shortening the entire sentence, changing it to "Ultimately, it allows us to place content over form and embrace the fluidity of the web and the diaspora of devices."

Reply:
Okie sir